polyfill/test/TimeZone/constructor/constructor/basic.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.timezone
+description: Basic tests for the Temporal.TimeZone constructor.
+features: [Temporal]
+---*/
+
+const valid = [
+  ["+01:00"],
+  ["-01:00"],
+  ["+0330", "+03:30"],
+  ["-0650", "-06:50"],
+  ["-08", "-08:00"],
+  ["\u221201:00", "-01:00"],
+  ["\u22120650", "-06:50"],
+  ["\u221208", "-08:00"],
+  ["+01:00:00", "+01:00"],
+  ["-010000", "-01:00"],
+  ["+03:30:00.000000001", "+03:30:00.000000001"],
+  ["-033000.1", "-03:30:00.1"],
+  ["Europe/Vienna"],
+  ["America/New_York"],
+  ["Africa/CAIRO", "Africa/Cairo"],
+  ["Asia/Ulan_Bator", "Asia/Ulaanbaatar"],
```

Comment:
I'm not sure, but wouldn't any tests involving non-offset, non-UTC time zones have to go in `intl402/`?